prime-router/src/testIntegration/resources/datatests/mapping-inventory/xtn/xtn_use_code_home.hl7

```diff
@@ -0,0 +1 @@
+MSH|^~\&#|STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|CDC Atlanta^11D0668319^CLIA|MEDSS-ELR ^2.16.840.1.114222.4.3.3.6.2.1^ISO|MNDOH^2.16.840.1.114222.4.1.3661^ISO|20230501102531-0400||ORU^R01^ORU_R01|3003786103_4988249_33033|T|2.5.1|||NE|NE|USA|UNICODE UTF-8|||PHLabReport-NoAck^PHIN^2.16.840.1.113883.9.11^ISO
PID|1||kc3oy^^^^PI^Any lab USA||Schaefer^Stephenie^Davis^^^^L||19930227|U||ASKU^Asked, but unknown^HL70005^^^^2.5.1|96633 Gilberto Course^^^IG||^PRN^BP^^1^260^7595016^^^^^+1 260 759 5016~^PRN^Internet^otto.daugherty@email.com|||||||||N^Non Hispanic or Latino^HL70189^^^^2.9||||||||N

```

Comment:
So one thing I'm struggling with looking at real tests, is I think it's going to be very difficult to remember what the functionality these files are specifically testing.  One way we could resolve that is adding anew column to config that is a description, but you could also move away from that config entirely and not use dynamic tests.